.github/workflows/eks-pratham-deployment.yaml

```
@@ -13,77 +12,77 @@ jobs:
-      env:
-       DOCKERFILE_FILE_CONTENT: ${{ secrets.DOCKERFILE_FILE_CONTENT }}
-      run: echo "$DOCKERFILE_FILE_CONTENT" > Dockerfile
-    - name: Show PWD and list content
-      run: |
+      - name: Set short git commit SHA
+        id: commit
+        uses: prompt/actions-commit-hash@v2
+      - name: Check out code
+        uses: actions/checkout@v2
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
```

actionlint

the runner of "aws-actions/configure-aws-credentials@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)